examples/auth/envoy_config.json

```diff
@@ -1,13 +1,12 @@
 {
   "admin": {},
   "node": {
-    "id": "ESPv2",
-    "cluster": "ESPv2_cluster"
+    "cluster": "ESPv2_cluster",
```

Comment:
How did you alphabetize the entries? Please add to the README in this directory

Reply:
I think `python -m json.tool` can alphabetize entries.